src/main/drivers/rx_pwm.c

```diff
@@ -385,12 +385,9 @@ void pwmRxInit(const pwmConfig_t *pwmConfig)
         IOInit(io, OWNER_PWMINPUT, RESOURCE_INDEX(channel));
 #ifdef STM32F1
         IOConfigGPIO(io, IOCFG_IPD);
-#elif defined(STM32F7)
```

Comment:
Why do we still need the #if #else construct here? The code looks to be the same. Do I miss anything?

Reply:
I think you are looking at the old code?

Reply:
Looks to be i was confused reading the code.